vortex-scalar/Cargo.toml

```diff
@@ -16,6 +16,7 @@ version = { workspace = true }
 [dependencies]
 arbitrary = { workspace = true, optional = true }
 arrow-array = { workspace = true }
+arrow-buffer = { workspace = true }
```

Comment:
Why is this needed? Should we use https://crates.io/crates/i256 instead?

Reply:
We should definitely re-export, don't mind which crate

Reply:
Discussed in Slack, think it's best to stick with the newtype wrapper like in the most recent commit on this branch. Can revert if you feel strongly though.

Reply:
I thought we would do a wrapper around i256 crate instead of the arrow one, like half

Reply:
Does it make a difference? it's obscured from the public API either way, and we care more about converting into/out of the Arrow i256?

Reply:
not sure if it matters. Would be nice if everyone used one bit representation.